File: Adam/prediction.py
infection = [64, 32, 16, 8, 4, 2, 1] #vcera, predevcirem atd.
AllR = 0
num_of_input_days = 7
days_to_predict = 3
optimistic_index = 0.9

for i in  range(num_of_input_days - 1):
    new = infection[i]
    old = infection[i + 1]
    R = -1

    if(old == 0):
        R = new
    elif(old == 0 and new == 0):
        R = 0
    else:
        R = new / old
    AllR += R

NumberR = AllR / (num_of_input_days - 1)

pessimistic_prediction = [infection[0] * NumberR]

for i in range(days_to_predict):
    pessimistic_prediction.append(pessimistic_prediction[i] * NumberR)

optimistic_prediction = [infection[0] * NumberR]

for i in range(days_to_predict):
    NumberR = NumberR * optimistic_index
    optimistic_prediction.append(optimistic_prediction[i] * NumberR)
    print(optimistic_prediction[i])

File: Vojta/Prediction.js
var infection = [8, 4, 2, 1, 0, 0, 0] //včera, předevčera atd.
var i;
var AllR = 0;
for (i = 0; i < infection.length-1; i++) {
    var New = infection[i];
    var Old = infection[i + 1];
    var R = -1.0;
    if (Old == 0) {
        R = New / 1;
    }
    else if (Old == 0 && New == 0) {
        R = 0;
    }
    else {
         R = New / Old;
    }
    AllR += R;
   
}
var NumberR = AllR / 7;
console.log(NumberR);
console.log('Hello world');   